decidim-core/app/views/layouts/decidim/header/_mobile_language_choose.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if available_locales.length > 1 %>
+    <div>
+        <button id="dropdown-trigger-language-chooser-mobile" data-component="dropdown" data-target="dropdown-menu-language-chooser-mobile" class="mt-8 border border-black rounded-lg py-0 bg-gray-5">
+            <span><%= t("name", scope: "locale" ) %></span>
+            <%= icon "arrow-down-s-line" %>
+            <%= icon "arrow-up-s-line" %>
+            <span class="sr-only">
+                <% available_locales.each do |locale| %>
+                <span lang="<%= locale %>">
+                    <%= I18n.with_locale(locale) { t("layouts.decidim.language_chooser.choose_language") } %>
+                </span>
+                <% end %>
+            </span>
+    </button>
+    <div id="dropdown-menu-language-chooser-mobile" class="bg-gray-7 relative" aria-hidden="true">
```

Comment:
As I mentioned in my other comment, we should think twice before adding more colors. Can we try this with a bg-gray-5 🙏🏽 ? 
```suggestion
    <div id="dropdown-menu-language-chooser-mobile" class="bg-gray-5 relative" aria-hidden="true">
```